resources/views/livewire/layout/navigation.blade.php

```diff
@@ -10,33 +10,40 @@
 
 ?>
 
-<nav x-data="{ open: false }" class="bg-white border-b border-gray-100">
+<nav x-data="{ open: false }" class="bg-radiation border-b border-radiation mb-0">
     <!-- Primary Navigation Menu -->
-    <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+    <div class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-7">
         <div class="flex justify-between h-16">
             <div class="flex">
                 <!-- Logo -->
                 <div class="shrink-0 flex items-center">
-                    <a href="{{ route('dashboard') }}" wire:navigate>
-                        <x-application-logo class="block h-9 w-auto fill-current text-gray-800" />
+                    <a href="{{ route('Principal') }}" wire:navigate>
```

Comment:
Essa rota não existe, mande para a rota "welcome" ou simplesmente adicione "/"
```suggestion
                    <a href="/" wire:navigate>
```